phy/cluster/algorithms.py

```diff
@@ -155,7 +164,7 @@ def __init__(self, tempdir=None, probe=None, **kwargs):
     def _create_filter(self):
         rate = self._kwargs['sample_rate']
         low = self._kwargs['filter_low']
```

Comment:
I agree - both should be in Hz.

For the .low.kwd - I don't think we need to extract LFP at the time of spike detection. We should be able to do this later, at the time of analysis.